### ISSUE TITLE
Release to npm

### ISSUE THREAD

**Issue by KittyGiraudel** [HugoGiraudel/node-legofy#6]
Don’t forget `.npmignore`.

**Comment by HugoGiraudel** [comment]
As soon as #17 is merged.

**Comment by valeriangalliat** [comment]
So publish as `node-legofy`? As `legofy`'s already taken.

**Comment by KittyGiraudel** [comment]
When trying to run the tests (prepublish):

```
> mocha test --compilers js:babel-core/register



  The Legofy module
    1) "before all" hook


  0 passing (318ms)
  1 failing

  1) The Legofy module "before all" hook:
     TypeError: undefined is not a function
      at legofy (index.js:64:26)
      at Context.<anonymous> (index.js:17:7)
```

Apparently related to `Object.assign`. Any idea @valeriangalliat?

**Comment by valeriangalliat** [comment]
Fixed with 65fee86980f90fd63a238ee3723bc1d1dbbf1fad.
